dev/TabView/TabView.cpp

```diff
@@ -428,3 +444,19 @@ winrt::DependencyObject TabView::ContainerFromIndex(int index)
     }
     return nullptr;
 }
+
+void TabView::OnCtrlF4Invoked(const winrt::KeyboardAccelerator& sender, const winrt::KeyboardAcceleratorInvokedEventArgs& args)
+{
+    if (IsEnabled())
```

Comment:
>IsEnabled [](start = 8, length = 9)

KeyboardAccelerators should only fire their invoked event if the element is Enabled - so I don't think you need this check.  Double-check that, but hopefully you can delete this check too to further simplify.

Reply:
Yep, removed.